apps/kahoot-it-clone/src/app/quiz/quiz-bank/quiz-util.ts

```diff
@@ -0,0 +1,40 @@
+import { FormBuilder, Validators, FormControl } from '@angular/forms';
+import { Shape } from '@kahoot-it-clone/shared-types';
+
+const fb = new FormBuilder();
+
+export const createQuestionGroup = () => {
+  return fb.group({
+    question: new FormControl('', Validators.required),
+    timeLimit: new FormControl(0, Validators.required),
+    points: new FormControl(0, Validators.required),
+    answerOptions: new FormControl('', Validators.required),
+    image: '',
+    answers: fb.array([
+      fb.group({
```

Comment:
You could probably shorten this file up by making another function that generates a FormGroups for each shape and returns a FormArray and then pass that FormArray into the answers variable on line 13